### PR TITLE
Adding ES troubleshooting entry

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

Adds troubleshooting instructions for Elasticsearch service errors caused by insufficient virtual memory allocation in the `vm.max_map_count` system configuration setting. This error can occur when launching a Magento Cloud Docker environment on Linux systems.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/docker/docker-containers-service.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/docker/docker-containers-service.md [hunk 1]
```diff
@@ -56,6 +56,23 @@ See [Manage the database] for details about using the database.
...
+### Troubleshooting
+
+On some Linux systems Elasticsearch container may not start with the following error:
+
+```bash
+ERROR: [1] bootstrap checks failed
+[1]: max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
+```
+
+To fix the error, adjust next value on the host machine:
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:68]:
```suggestion
To fix the error, run the following `sysctl` command to increase the memory map area allocation.
```

File: src/cloud/docker/docker-containers-service.md [hunk 2]
```diff
@@ -56,6 +56,23 @@ See [Manage the database] for details about using the database.
...
+
+```bash
+sysctl -w vm.max_map_count=262144
+```
+
+To set this value permanently, update the `vm.max_map_count` setting in `/etc/sysctl.conf`. To verify after rebooting, run `sysctl vm.max_map_count`.
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:74]:
```suggestion
{:.procedure}
To permanently update the sysctl value for `vm.max_map_count`:

1. In the sysctl configuration file (`etc/sysctl.conf`), change the value for the `vm.max_map_count` option to the required setting.

1. Reboot your system.

1. Run the following command to verify the change.

   ```bash
   sysctl vm.max_map_count
   ```
```

File: src/cloud/docker/docker-containers-service.md [hunk 3]
```diff
@@ -56,6 +56,23 @@ See [Manage the database] for details about using the database.
 
 The Elasticsearch container for {{site.data.var.mcd-prod}} is a standard Elasticsearch container with required plugins and configurations for {{site.data.var.ee}}.
 
+### Troubleshooting
+
+On some Linux systems Elasticsearch container may not start with the following error:
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:61]:
```suggestion
On some Linux systems, when you launch the Docker environment, the Elasticsearch service fails to start and the following error displays:
```

File: src/cloud/docker/docker-containers-service.md [hunk 4]
```diff
@@ -56,6 +56,33 @@ See [Manage the database] for details about using the database.
...
+To permanently update the system setting for `vm.max_map_count`:
+
+1. Edit the sysctl configuration file (`etc/sysctl.conf`) and set the required value for the `vm.max_map_count` option.
+
+1. Reboot your system.
+
+1. Run the following command to verify the change.
+
+   ```bash
+   sysctl vm.max_map_count
```
Comment (meker12) [inline src/cloud/docker/docker-containers-service.md:84]:
```suggestion
   sysctl vm.max_map_count
   ```
```

**Comment by meker12** [comment]
running tests

**Pull request merged**